theme/styles/settings/_z-index.scss

```diff
@@ -4,7 +4,8 @@ $z: (
   '1': 1,
```

Comment:
Curious about these values. They seem pretty arbitrary, and also combine explicit (z-1) with descriptive (z-nav) naming. I say we choose one. Also, maybe we stick to increments of ten? z-0, z-10, z-20, z-30 and so on.

Reply:
Gooood point! I'm closing this PR for now because Knowledge Areas will be on hold until a future version, but will update z-index in another PR. :)